noxfile.py

```diff
@@ -183,6 +183,75 @@ def system(session):
         )
 
 
+@nox.session(python=DEFAULT_PYTHON_VERSION)
+def prerelease(session):
```

Comment:
Note: Mostly taken straight from https://github.com/googleapis/python-bigquery-pandas/blob/d4539747848156446d8a3ccdb49661e72ae03109/noxfile.py#L176

https://github.com/googleapis/synthtool/issues/1290 is open to add this to the templates, though it would need to be slightly templated to allow for customized prerelease dependencies.